fw/sock.c

```diff
@@ -1498,7 +1498,11 @@ static inline void
 ss_do_shutdown(struct sock *sk)
```

Comment:
Please update the copy right year - check you .git configuration, it should be done in automatic way on commit